README.MD

```diff
@@ -2,3 +2,73 @@
 
 The `libobmcsession` library aims to provide a common way to manage OpenBMC user sessions of any subsystem in which an authorization feature is used.
 
+## Build with OpenBMC SDK
+OpenBMC SDK contains toolchain and all dependencies needed for building the
+project. See [official
+documentation](https://github.com/openbmc/docs/blob/master/development/dev-environment.md#download-and-install-sdk)
+for details.
+
+Build steps:
+```sh
+$ source /path/to/sdk/environment-setup-arm1176jzs-openbmc-linux-gnueabi
+$ meson --buildtype plain --optimization s build_dir
+$ ninja -C build_dir
+```
+If build process succeeded, the directory `build_dir` contains executable file
+`libobmcsession.so`.
+
+## Design
+
```

Comment:
Do you plan to inject this library into each service that the user communicate with?

Reply:
Yes, this is the goal the `libobmcsession` developed on.